src/Components/reusable-components/ThreadCardComponents/ThreadLeftComponentText.js

```diff
@@ -9,7 +9,7 @@ function ThreadLeftComponentText(props) {
       <div className="top">
         <div className="bold">{createdBy} started a thread</div>
         <div className="lighter-color">
-          {createdAt} in {space}
+          {createdAt} {space}
         </div>
```

Comment:
add an  in

Reply:
maybe even an started thread at?

Reply:
@samarv confirmed that we are getting rid of the 'in' for now because it doesn't currently show which space the thread is for in the homescreen, and 'in' isn't necessary when viewing the threads within a space either.

Reply:
@ThorbenBender